Makefile

```diff
@@ -209,6 +222,49 @@ $(BUILD_DIRECTORY)/electron-$(ELECTRON_VERSION)-$(TARGET_PLATFORM)-$(TARGET_ARCH
 		-s "$(TARGET_PLATFORM)" \
 		-o $@
 
+$(BUILD_DIRECTORY)/$(APPLICATION_NAME)-cli-$(TARGET_PLATFORM)-$(APPLICATION_VERSION)-$(TARGET_ARCH)-app: \
+	package.json lib \
+	$(BUILD_DIRECTORY)/node-$(TARGET_PLATFORM)-$(TARGET_ARCH)-dependencies/node_modules \
+	| $(BUILD_DIRECTORY)
+	mkdir $@
+	$(foreach prerequisite,$^,$(call execute-command,cp -rf $(prerequisite) $@))
+
+$(BUILD_DIRECTORY)/$(APPLICATION_NAME)-cli-$(TARGET_PLATFORM)-$(APPLICATION_VERSION)-$(TARGET_ARCH).js: \
+	$(BUILD_DIRECTORY)/$(APPLICATION_NAME)-cli-$(TARGET_PLATFORM)-$(APPLICATION_VERSION)-$(TARGET_ARCH)-app \
+	| $(BUILD_DIRECTORY)
+	./scripts/build/concatenate-javascript.sh -e $</lib/cli/etcher.js -o $@
+
+$(BUILD_DIRECTORY)/$(APPLICATION_NAME)-cli-$(APPLICATION_VERSION)-$(TARGET_PLATFORM)-$(TARGET_ARCH): \
+	$(BUILD_DIRECTORY)/node-$(TARGET_PLATFORM)-$(TARGET_ARCH)-dependencies/node_modules \
+	$(BUILD_DIRECTORY)/$(APPLICATION_NAME)-cli-$(TARGET_PLATFORM)-$(APPLICATION_VERSION)-$(TARGET_ARCH).js \
+	| $(BUILD_DIRECTORY) $(BUILD_TEMPORARY_DIRECTORY)
+	./scripts/build/node-package-cli.sh -o $@ -l $< \
+		-n $(APPLICATION_NAME) \
+		-e $(word 2,$^) \
+		-r $(TARGET_ARCH) \
+		-s $(TARGET_PLATFORM) \
+		-d "$(APPLICATION_DESCRIPTION)" \
+		-v "$(APPLICATION_VERSION)" \
+		-c "$(APPLICATION_COPYRIGHT)" \
+		-m "$(COMPANY_NAME)" \
+		-i assets/icon.ico \
+		-w $(BUILD_TEMPORARY_DIRECTORY)
+ifeq ($(TARGET_PLATFORM),darwin)
```

Comment:
Done, I added spaces between each platform specific block to make it easier to the eye. Not sure what to put on a comment here though. What did you have in mind?